app/presenters/nsm/check_answers/cost_summary_card.rb

```diff
@@ -67,6 +67,12 @@ def total_gross_allowed
         sum_allowed(data_for_footer, :allowed_gross_cost)
       end
 
+      def total_gross_adjusted?
+        return false unless claim.granted? || claim.part_grant?
```

Comment:
would it be more SRP if we move that guard out and its that something we check separately before hitting `total_gross_adjusted?` ie  `return "#{NumberTo.pounds(total_gross_allowed)} allowed" if not_granted_or_part_grant? && total_gross_adjusted?`?

Reply:
This could be because it is not really named appropriately. It could/should be named `show_adjustments?`. It is being used to display (or not) the adjusted total and the adjusted tab with a breakdown of the adjusted values. 

Would that make sense?

Reply:
`show_adjustments?` feels clearer 😄 - cheers!

Reply:
unfortunately `show_adjustments` is a var heavily used in the cost summary card and elsewhere. need another name that makes sense really. 🤔

Reply:
`display_adjustments?`